subprojects/test-runner/client/src/main/kotlin/com/avito/runner/scheduler/metrics/TestMetricsSender.kt

```diff
@@ -34,6 +34,6 @@ internal class TestMetricsSender(
     }
 
     fun sendMedianDeviceUtilization(percent: Int) {
-        statsDSender.send(GaugeLongMetric(prefix.append("device-utilization.median"), percent.toLong()))
```

Comment:
This name has been converted to `device-utilization_median` due to inner `SeriesName` logics